kubectx

```diff
@@ -185,6 +186,11 @@ delete_context() {
   $KUBECTL config delete-context "${ctx}"
 }
 
+unset_context() {
+  echo "Unsetting default context." >&2
```

Comment:
default -> current

Reply:
I think this message is not really necessary, as kubectl will always print  `Property "current-context" unset.` even though the file is empty.

```
$ KUBECONFIG=$(mktemp) kubectl config unset current-context
Property "current-context" unset.
```

So the "progress" message is not necessary.

Reply:
Just going for consistency. Eg.

```
kubectx -d gke_dl-rapid_us-central1-a_rapid-dev
Deleting context "gke_dl-rapid_us-central1-a_rapid-dev"...
deleted context gke_dl-rapid_us-central1-a_rapid-dev from /Users/rsa/.kube/config
```